components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt

```diff
@@ -15,27 +16,22 @@ import android.widget.RelativeLayout
 import android.widget.TextView
 import androidx.annotation.ColorInt
 import androidx.core.content.ContextCompat
-import mozilla.components.support.ktx.android.view.onNextGlobalLayout
 import mozilla.components.support.utils.DrawableUtils
 import java.text.NumberFormat
 
-@Suppress("LargeClass")
-open class TabCounter @JvmOverloads constructor(
+class TabCounter @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyle: Int = 0
 ) : RelativeLayout(context, attrs, defStyle) {
 
     private val box: ImageView
-    private val bar: ImageView
```

Comment:
We don't want the PR to be about removing the bar necessarily since it should be about fixing the tab count. So, I would suggest undo-ing some of these changes and just make sure it is refined to the problem we are trying to solve. We could introduce changes to remove the bar in a different commit/PR.

Reply:
Just the bar for now need undo-ing or is there another?

Reply:
I would probably undo most changes that does not touch the count. The PR should look similar to your limited changes on resolving the count like your Fenix PR.

Reply:
I'm gonna close this pr and open a new pr with just the logic? Is this okey?

Reply:
Yep, organize it however you want to.